### PR TITLE
Revise hardhat to use Ignition modules

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Revises Hardhat Tutorial to use Ignition modules and migrates screenshots over to Terminal snippets

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

**Review by eshaben — COMMENTED** [review]
Awesome! Can you please check out my comments on the Moonbeam PR and apply feedback here too? 🙂

**Comment by themacexpert** [comment]
> Awesome! Can you please check out my comments on the Moonbeam PR and apply feedback here too? 🙂

Done

File: dapp-developers/developer-toolkit/ethereum-api/dev-env/.pages [hunk 1]
```diff
@@ -2,6 +2,6 @@ title: Dev Environments
 hide: false
 nav:
   - index.md
-  - remix.md
-  - hardhat.md
-  - foundry.md
+  - 'Deploy Contracts with Remix': remix.md
+  - 'Deploy Contracts with Hardhat': hardhat.md
+  - 'Deploy Contracts with Foundry': foundry.md
```
Comment (eshaben) [inline dapp-developers/developer-toolkit/ethereum-api/dev-env/.pages:7]: thank you for doing this!! i guess i didn't even realize that these were all using "Deploy Contracts with X" in the left nav- can you please change them to be "Remix", "Hardhat", and "Foundry"? so it aligns with the rest of the sections, where we just list the tool name

**Review by eshaben — COMMENTED** [review]
Beautiful!!

**Review by eshaben — APPROVED** [review]
LGTM! Thanks!! 🚀

**Pull request merged**